src/main/java/com/google/devtools/build/docgen/templates/be/make-variables.vm

```diff
@@ -74,6 +74,17 @@ title: Make Variables
   rule, Bazel fails with an error.
 </p>
 
+
+<p>
+  "Make" variables whose names are non-letter symbols, such as
```

Comment:
The question is if we should emphasize this, or should we be moving to removed the special case for this. I'm not sure today.

Reply:
I'd equally happily accept a bug/issue regarding such a deprecation & migration.
Honestly I hate the obscure symbol-only names of these variables in general.